lib/components/util/transit-operator-icons.tsx

```diff
@@ -0,0 +1,88 @@
+import { MapPin } from '@styled-icons/fa-solid'
+import { useIntl } from 'react-intl'
+import React from 'react'
+import Skeleton from 'react-loading-skeleton'
+import type { TransitOperator } from '@opentripplanner/types'
+
+import InvisibleA11yLabel from './invisible-a11y-label'
+import OperatorLogo from './operator-logo'
+import type { StopData } from './types'
+
+const Operator = ({ operator }: { operator?: TransitOperator }) => {
+  const intl = useIntl()
+
+  if (!operator) {
+    return null
+  } else {
+    const operatorLogoAriaLabel = intl.formatMessage(
+      {
+        id: 'components.StopViewer.operatorLogoAriaLabel'
+      },
+      {
+        operatorName: operator.name
+      }
+    )
+    return operator.logo ? (
+      // Span with agency classname allows optional contrast/customization in user
+      // config for logos with poor contrast. Class name is hyphenated agency name
+      // e.g. "sound-transit"
+      <span
+        className={
+          operator.name ? operator.name.replace(/\s+/g, '-').toLowerCase() : ''
+        }
+      >
+        <OperatorLogo
+          alt={operatorLogoAriaLabel}
+          marginRight={0.5}
+          maxHeight={1}
+          operator={operator}
+        />
+      </span>
+    ) : (
+      // If operator exists but logo is missing,
+      // we still need to announce the operator name to screen readers.
+      <>
+        <MapPin />
+        <InvisibleA11yLabel>{operatorLogoAriaLabel}</InvisibleA11yLabel>
+      </>
+    )
+  }
+}
+
+const TransitOperatorLogos = ({
+  loading = false,
+  stopData,
+  transitOperators
+}: {
+  loading?: boolean
+  stopData: StopData | undefined
```

Comment:
Use `?:` to denote that a prop can also be `undefined`.

Reply:
🤦‍♀️